### PR TITLE
Feed posts

### PR DESCRIPTION
**Pull request by lisset-orozco**
- [x]  Pagination of posts

- [x] Infinite scroll with will_paginate

- [x] Enable ActionCable, use Redis, and create a channel for posts

- [x] Real-time posts for all

- [x] Identify the current user in ActionCable using cookies with warden_hooks

- [x] Real-time posts for only friends

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -13,10 +13,31 @@
...
         .or(Post.where(user_id: user.friend_ids))
         .or(Post.where(user_id: user.user_ids))
   end
+
+  private
+
+  def send_to_action_cable
+    data = { message: to_html, action: 'new_post' }
+
+    user.friend_ids.each do |friend_id|
```
Comment (joalbertg) [inline app/models/post.rb]: refactor to a single line with `{...}` and create a new method for the repeated content

File: app/models/post.rb [hunk 2]
```diff
@@ -13,10 +13,31 @@
...
+      ActionCable.server.broadcast "posts_#{friend_id}", data
+    end
+
+    user.user_ids.each do |friend_id|
```
Comment (joalbertg) [inline app/models/post.rb]: same here

**Review by joalbertg — CHANGES_REQUESTED** [review]
Very good!

File: app/models/post.rb (hunk 1 above)
Comment (lisset-orozco) [reply app/models/post.rb]: Done!

File: app/models/post.rb (hunk 2 above)
Comment (lisset-orozco) [reply app/models/post.rb]: Done!

**Review by joalbertg — APPROVED** [review]

**Review by joalbertg — APPROVED** [review]

**Pull request merged**